docs/_artifacts/document-attributes.adoc

```diff
@@ -3,41 +3,41 @@
 // Attributes MUST appear below the document title
 // Do not delete attributes with identical values, such as :product-long: and :product:. These are intentional because although they are identical here, they are not in other renderings of this source file.
 
-
+//General
 :org-name: Application Services
 :community:
 :imagesdir: ./images
-:product-version: 1
-:product-long: OpenShift Streams for Apache Kafka
-:product: Streams for Apache Kafka
-:registry-product-long: OpenShift Service Registry
-:registry-product: Service Registry
-:registry: Service Registry
 :cli-product-long: OpenShift Application Services
-// Placeholder URL, when we get a HOST UI for the service we can put it here properly
-:service-url: https://console.redhat.com/application-services/streams/
-:registry-url: https://console.redhat.com/application-services/service-registry/
 :property-file-name: app-services.properties
-
-// Other upstream project names
 :samples-git-repo: https://github.com/redhat-developer/app-services-guides
-
-//URL components for cross refs
-:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/docs/
 :base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
-:getting-started-url: getting-started/README.adoc
-:getting-started-service-registry-url: getting-started-service-registry/README.adoc
-:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
-:kafkacat-url: kcat/README.adoc
-:quarkus-url: quarkus/README.adoc
-:quarkus-service-registry-url: quarkus-service-registry/README.adoc
 :rhoas-cli-url: rhoas-cli/README.adoc
-:rhoas-cli-kafka-url: rhoas-cli-kafka/README.adoc
-:rhoas-cli-service-registry-url: rhoas-cli-service-registry/README.adoc
 :rhoas-cli-ref-url: commands
-:topic-config-url: topic-configuration/README.adoc
-:consumer-config-url: consumer-configuration/README.adoc
 :service-binding-url: service-binding/README.adoc
```

Comment:
SBO is included in both so we can move it to the folder and adjust that file:
https://github.com/redhat-developer/app-services-guides/blob/main/docs/.product-mapping

Reply:
@wtrocki @jbyrne-redhat ack. For now then, I've moved service-binding into kafka specifically, and renamed the attribute for it to also be for kafka specifically, then when John submits his RHOSR-specific version, can think again on whether to do a segregated or single-sourced approach.

Reply:
@jbyrne-redhat Maybe prefix one with "kafka-" and the other with "service-registry"?

Reply:
Agreed @bredamc , but I suppose a suffix (or affix) instead of a prefix just because that's been the norm in the guide names so far from what I've seen in the others. So `service-binding-kafka-url` and `service-binding-service-registry-url`. The product names use prefixes though, so we're not totally consistent across the board right now. Maybe it makes sense to continue to prefix prod names to highlight which prod and continue to affix the guide names to highlight the guide? IDK, just a thought.